masterlist.yaml

```
@@ -5081,6 +5081,15 @@ plugins:
         subs: [ 'Regulators ESP' ]
         condition: 'many("Regulators(_Overhaul(_lvl(2|7))?)?\.esp")'
 
+  - name: 'There's No Such Thing As Bad Publicity.esp'
```

single `'` need to be escaped with amother `'`.
```suggestion
  - name: 'There''s No Such Thing As Bad Publicity.esp'
```